app-misc/cadubi/cadubi-1.3.3.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="An application that allows you to draw ASCII-Art images"
+HOMEPAGE="https://github.com/statico/cadubi"
+SRC_URI="https://github.com/statico/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Artistic"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+
+DEPEND="dev-lang/perl
```

Comment:
Shouldn't be this ```RDEPEND```? Based on upstreams repo, this seems only normal perl scripts without any compilation involved?